tests/pytest/eligibility/test_forms.py

```diff
@@ -1,4 +1,4 @@
-from benefits.eligibility.forms import CSTAgencyCard, MSTCourtesyCard, SBMTDMobilityPass
```

Comment:
I think we should keep these tests, as we do still need the form classes and we can assert that they are set up correctly.

Reply:
Reverted that last commit: https://github.com/cal-itp/benefits/pull/2214/commits/e8838b21bdbc5f9711c488f4eb291cf220643279